maestro-cli/src/main/java/maestro/cli/report/JUnitTestSuiteReporter.kt

```diff
@@ -47,7 +47,8 @@ class JUnitTestSuiteReporter(
                                                 Failure(
                                                     message = failure.message,
                                                 )
-                                            }
+                                            },
+                                            time = flow.duration?.inWholeMilliseconds?.let{ (it / 1000.0).toString() }
```

Comment:
Please round this to whole seconds to ensure it is consistent with how it is logged in the console. When testing this locally, the report contains duration on a millisecond granularity:

```
<testcase id="android-advanced-flow" name="android-advanced-flow" classname="android-advanced-flow" time="18.045" />
```